Dockerfile

```diff
@@ -1,23 +1,50 @@
-FROM python:2.7
-ENV PYTHONUNBUFFERED 1
-ENV PYTHONPATH /app
+FROM ubuntu:focal as app
```

Comment:
I used this `Dockerfile` to run tests locally, as notably, the app requires a PostreSQL installation.

This could be excluded from the PR, though I did find it helpful.
I'm admittedly not sure what the previous `Dockerfile` was trying to do...

This file was cribbed from another edx repo (discovery or registrar; I forget now).